Backport notes to changelog (#4407)
mattermost/docs: source/administration/changelog.md

## Release v5.33 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)

- **v5.33.2, release day TBD**

>   - Improving typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)

Reviewer: Oops, I really messed up the grammar here! Sorry:
```suggestion
  - Improved typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```

Author: This can be changed once the dot release has been shipped.

>   - Fixing performance issues with the new sidebar that affects typing performance on busy servers. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)

Reviewer: My only concern with the term `fixing` is that it implies a complete resolution, whereas I think there is still substantial improvements to be made.

0/5, but I wonder if we should phrase as:
```suggestion
  - Improving typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```

## Release v5.31 - [ESR](https://docs.mattermost.com/administration/extended-support-release.html)

- **v5.31.2, release day TBD**

>   - Improving typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)

Reviewer: ```suggestion
  - Improved typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```
